docs/installation/windows.md

```diff
@@ -97,53 +97,28 @@ Pay attention to the username and password you setup during installation of Post
 
 ## Installing Dev.to
 
-1.  Fork dev.to repository, ie. https://github.com/thepracticaldev/dev.to/fork
-2.  Clone your forked repository, ie. `git clone https://github.com/<your-username>/dev.to.git`
-3.  Install bundler with `gem install bundler`
-4.  Install foreman gem with `gem install foreman`
-5.  Run `rbenv rehash` when foreman installation is done.
-6.  Install our ruby dependencies with `bundle install`
-    While installing, you might run into an error due to the pg gem requiring PostgreSQL libraries. If so, please run `sudo apt-get install libpq-dev` before retrying.
-7.  Setup your database
+1. Fork dev.to repository, ie. https://github.com/thepracticaldev/dev.to/fork
+1. Clone your forked repository, ie. `git clone https://github.com/<your-username>/dev.to.git`
```

Comment:
the points became all 1s hehehe :D

Reply:
Markdown converts all the 1s to accurate numeric values 💪

Reply:
oooooooh didn't know that! thanks, you made my day!!!!

Reply:
It mostly works but you have to remember to indent anything in between the numbered lines.